src/Statistics/Repositories/TopScoreRepository.php

```diff
@@ -0,0 +1,13 @@
+<?php declare(strict_types=1);
+
+namespace VSV\GVQ_API\Statistics\Repositories;
+
+use VSV\GVQ_API\Statistics\TopScore;
+use VSV\GVQ_API\User\ValueObjects\Email;
+
+interface TopScoreRepository
+{
+    public function findByEmail(Email $email): ?TopScore;
+
+    public function set(TopScore $topScore): void;
```

Comment:
Voor het bewaren gebruiken we als naam `save` omdat het geen key/value object is maar een model object.